src/cozmo/robot.py

```diff
@@ -130,6 +130,67 @@ def _encode(self):
                                                 useManualSpeed=False,
                                                 usePreDockPose=False)
 
+class DockWithCube(action.Action):
+    '''Represents the dock with cube action in progress.
+
+    Returned by :meth:`~cozmo.robot.Robot.dock_with_cube`
+    '''
+    _action_type = _clad_to_engine_cozmo.RobotActionType.ALIGN_WITH_OBJECT
+
+    def __init__(self, obj, approach_angle, **kw):
+        super().__init__(**kw)
+        #: The object (e.g. an instance of :class:`cozmo.objects.LightCube`) that is being put down
+        self.obj = obj
+        if approach_angle is None:
+            self.useApproachAngle = False
```

Comment:
python variables should be snake-cased, so should be self.use_approach_angle = False
and self.approach_angle =